tests/lax_numpy_test.py

```diff
@@ -2874,6 +2874,24 @@ def testPolyMul(self, a1_shape, a2_shape, dtype):
     self._CheckAgainstNumpy(np_fun, jnp_fun_np, args_maker, check_dtypes=False, tol=tol)
     self._CompileAndCheck(jnp_fun_co, args_maker, check_dtypes=False)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "a_shape={} , b_shape={}".format(
+          jtu.format_shape_dtype_string(a_shape, dtype),
+          jtu.format_shape_dtype_string(b_shape, dtype)),
+       "dtype": dtype, "a_shape": a_shape, "b_shape" : b_shape}
+      for dtype in default_dtypes
+      for a_shape in one_dim_array_shapes
+      for b_shape in one_dim_array_shapes))
+  def testPolyDiv(self, a_shape, b_shape, dtype):
+    rng = jtu.rand_default(self.rng())
+    np_fun = lambda arg1, arg2: np.polydiv(arg1, arg2)
+    jnp_fun = lambda arg1, arg2: jnp.polydiv(arg1, arg2)
```

Comment:
No need for lambdas, we can write `np_fun = np.polydiv` and `jnp_fun = jnp.polydiv`, or just use `np.polydiv` and `jnp.polydiv` directly below.